### PR TITLE
doc: Add antenna connectors

### PR DESCRIPTION
**Pull request by pradeeka7**
#### Summary

Antenna Connectors tutorial for The Things Advanced certification.
Objectives: Recognizing antenna connectors with their male/female/reverse polarity (if have) variations.

#### Screenshots

image

#### Changes

- ...
- ...

#### Notes for Reviewers

@benolayinka and Orkhan AmirAslan

#### Checklist

- [X] Scope: The referenced issue is addressed, there are no unrelated changes.
- [X] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links.
- [X] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [X] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/lorawan/antenna-connectors/_index.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+| {{< figure src="00662-1.jpg" alt="PR-SMA to U.FL" >}}    | Interface Cable: RP-SMA to U.FL. _Image courtesy of [SparkFun Electronics](https://www.sparkfun.com/)_        |
+| {{< figure src="rubber-ducky-antenna-with-SMA-connector.png" alt="Rubber ducky antenna with SMA" >}}    | An omni-directional rubber ducky antenna with SMA connector. |
+
+## N Type
+
+N Type connectors are threaded, waterproof, medium-size RF connectors that can be used to connect antennas (LoRa, LTE, GPS, WiFi, etc) directly or using cables.
+
+There are two types:
+
+- **N Type Male** - these connectors have a center pin and inner threads. There is a ferrule exists between the center pin and the inner threads.
```
Comment (nejraselimovic) [inline doc/content/lorawan/antenna-connectors/_index.md]:
```suggestion
- **N Type Male** - these connectors have a center pin and inner threads. There is a ferrule between the center pin and the inner threads.
```

File: doc/content/lorawan/antenna-connectors/_index.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+- **N Type Female** - these connectors have a center sleeve and outer threads.
+
+{{< figure src="N-type-female.png" alt="N Type Female" >}}
+
+**Variation:** Like SMA connectors, N Type connectors have reverse polarity variations known as RP-N Type connectors.
+
+There are two types:
+
+- **RP-N Male** -  these connectors have a center sleeve and inner threads. There is a ferrule exists between the center pin and the inner threads.
```
Comment (nejraselimovic) [inline doc/content/lorawan/antenna-connectors/_index.md]:
```suggestion

- **RP-N Male** -  these connectors have a center sleeve and inner threads. There is a ferrule between the center pin and the inner threads.
```

File: doc/content/lorawan/antenna-connectors/troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+title: "Troubleshooting Antenna Connectors"
```
Comment (nejraselimovic) [inline doc/content/lorawan/antenna-connectors/troubleshooting.md]: what is this?

**Review by nejraselimovic — CHANGES_REQUESTED** [review]

File: doc/content/lorawan/antenna-connectors/troubleshooting.md (hunk 3 above)
Comment (pradeeka7) [reply doc/content/lorawan/antenna-connectors/troubleshooting.md]: It is an auto-generated file. I will delete it.

**Comment by pradeeka7** [comment]
@nejraselimovic done

**Pull request merged**